pkg/compare/compare.go

```diff
@@ -96,12 +99,21 @@ const (
 	DiffSeparator            = "**********************************"
 )
 
+const (
+	Json  string = "json"
+	Yaml         = "yaml"
+	Human        = "human"
```

Comment:
in this case "human" would be default output (what we currently have)?

Reply:
yes

Reply:
Gotcha! So in that case can we treat this as "default" and just drop reference to "human"? It sort of reads like we are creating a new file type `.human` (like .json + .yaml)?

Reply:
not sure i understood exactly your suggestion , but i added a default option that is equal to the human one